harness/determined/cli/job.py

```diff
@@ -167,6 +227,21 @@ def update(args: Namespace) -> None:
                     ),
                 ],
             ),
+            Cmd(
+                "update-batch",
+                process_updates,
+                "batch update jobs",
+                [
+                    Arg(
+                        "operation",
+                        nargs=ONE_OR_MORE,
```

Comment:
cool! I did not know about this. argparse docs don't even mention `ONE_OR_MORE` — they use `nargs="+"` directly.